tables/guardduty_finding/guardduty_finding.go

```diff
@@ -0,0 +1,141 @@
+package guardduty_finding
+
+import (
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/service/guardduty/types"
+	"github.com/turbot/tailpipe-plugin-sdk/schema"
+)
+
+type GuardDutyFinding struct {
+	schema.CommonFields
+
+	AccountId     *string                   `json:"account_id"`
+	Arn           *string                   `json:"arn"`
+	Description   *string                   `json:"description"`
+	Id            *string                   `json:"id"`
+	Partition     *string                   `json:"partition"`
+	Region        *string                   `json:"region"`
+	SchemaVersion *string                   `json:"schema_version"`
+	Severity      *float64                  `json:"severity"`
+	Title         *string                   `json:"title"`
+	Type          *string                   `json:"type"`
+	CreatedAt     time.Time                 `json:"created_at"`
+	UpdatedAt     *time.Time                `json:"updated_at"`
+	Service       *Service                  `json:"service,omitempty"`
+	Resource      *GuardDutyFindingResource `json:"resource,omitempty"`
+}
+
+type Service struct {
+	Action               *GuardDutyFindingAction              `json:"action,omitempty"`
+	AdditionalInfo       *GuardDutyFindingServiceAdditionInfo `json:"additional_info,omitempty"`
+	Archived             *bool                                `json:"archived,omitempty"`
+	Count                *int32                               `json:"count,omitempty"`
+	Detection            *types.Detection                     `json:"detection,omitempty" parquet:"type=JSON"` // contains maps
+	DetectorId           *string                              `json:"detector_id,omitempty"`
+	EbsVolumeScanDetails *types.EbsVolumeScanDetails          `json:"ebs_volume_scan_details,omitempty" parquet:"type=JSON"` // contains []struct
+	EventFirstSeen       *string                              `json:"event_first_seen,omitempty"`
+	EventLastSeen        *string                              `json:"event_last_seen,omitempty"`
+	Evidence             *types.Evidence                      `json:"evidence,omitempty" parquet:"type=JSON"` // contains []struct
+	FeatureName          *string                              `json:"feature_name,omitempty"`
+	MalwareScanDetails   *types.MalwareScanDetails            `json:"malware_scan_details,omitempty" parquet:"type=JSON"` // contains []struct
+	ResourceRole         *string                              `json:"resource_role,omitempty"`
+	RuntimeDetails       *GuardDutyFindingRuntimeDetails      `json:"runtime_details,omitempty"`
+	ServiceName          *string                              `json:"service_name,omitempty"`
+	UserFeedback         *string                              `json:"user_feedback,omitempty"`
+}
+
+type GuardDutyFindingServiceAdditionInfo struct {
```

Comment:
```suggestion
type ServiceAdditionalInfo struct {
```

For this struct and all others below it, I don't think we need the `GuardDutyFinding` prefix.